nsdb-cluster/src/main/scala/io/radicalbit/nsdb/cluster/logic/CapacityWriteNodesSelectionLogic.scala

```diff
@@ -0,0 +1,55 @@
+/*
+ * Copyright 2018 Radicalbit S.r.l.
```

Comment:
It isn't up to date @saveveltri

Reply:
yes. agreed. thanks for posting this issue. I think the copyright update deserves a dedicate PR.